language/move-binary-format/src/errors.rs

```diff
@@ -82,10 +82,17 @@ impl VMError {
                 VMStatus::Error(StatusCode::ABORTED)
             }
 
-            // TODO Errors for OUT_OF_GAS do not always have index set
             (major_status, sub_status, location)
                 if major_status.status_type() == StatusType::Execution =>
             {
+                let abort_location = match &location {
+                    Location::Script => vm_status::AbortLocation::Script,
+                    Location::Module(id) => vm_status::AbortLocation::Module(id.clone()),
+                    Location::Undefined => {
+                        return VMStatus::Error(major_status);
+                    }
+                };
+                // Errors for OUT_OF_GAS do not always have index set, it should already return above.
```

Comment:
This should read a bit tighter than that
```suggestion
                // Errors for OUT_OF_GAS do not always have index set: if it does not, it should already return above.
```